src/Common/src/Interop/Windows/sspicli/NegotiationInfoClass.cs

```diff
@@ -8,12 +8,8 @@ namespace System.Net
 {
     // This class is used to determine if NTLM or
     // Kerberos are used in the context of a Negotiate handshake
-    internal class NegotiationInfoClass
+    internal partial class NegotiationInfoClass
     {
-        internal const string NTLM = "NTLM";
```

Comment:
What was the reason this got separated in two? I find it odd to have a partial class only to define 3 const strings. Shouldn't we create a new class for these strings instead if they can't live here?